dev/com.ibm.ws.grpc.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.grpc.client/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+
+clientConfig=GRPC Client Properties
+clientConfig.desc=Configuration properties to be applied to gRPC targets that match the specified URI when they are constructed
```

Comment:
Does "when they are constructed" refer to the time that the configuration properties are applied or the time that gRPC targets  match the specified URI?

Reply:
This isn't phrased well at all - I used `com.ibm.ws.jaxrs.2.x.config/resources/OSGI-INF/l10n/metatype.properties` as a guide.  

I'd intended for "when they are constructed" to apply to "the time that gRPC targets match the specified URI".  Would removing it completely be more clear?
`Configuration properties to be applied to gRPC targets that match the specified URI`